configure.ac

```diff
@@ -160,12 +160,6 @@ AC_ARG_WITH([miniupnpc],
   [use_upnp=$withval],
   [use_upnp=auto])
 
-AC_ARG_WITH([natpmp],
```

Comment:
Commit a749ab8cfe `build: Drop libnatpmp from build system` leaves some mentions of "natpmp" in `configure.ac` which are later removed by 6f4bc73e39 `depends: Drop natpmp and associated option from depends`. I think all removals of "natpmp" from `configure.ac` belong to one commit.